wurst/file/SaveLoadData.wurst

```diff
@@ -0,0 +1,82 @@
+package SaveLoadData
+import FileIO
+import SyncSimple
+import ErrorHandling
+
+/* 	Comfort wrapper for player based, synced access to save data.
+	For small data you can use strings, otherwise use ChunkedString.
```

Comment:
Seems weird to push the responsibility on the user

Can you support one or the other?